packages/react/src/components/navigation/navigationLanguageSelector/NavigationLanguageSelector.tsx

```diff
@@ -3,9 +3,20 @@ import React from 'react';
 import styles from './NavigationLanguageSelector.module.scss';
 import { MenuButton, MenuButtonProps } from '../../../internal/menuButton/MenuButton';
 
-export const NavigationLanguageSelector = ({ children, id = 'languageSelector', label, ...rest }: MenuButtonProps) => (
-  <MenuButton className={styles.languageDropdown} id={id} label={label} menuOffset={10} {...rest} closeOnItemClick>
-    {children}
-  </MenuButton>
-);
+export const NavigationLanguageSelector = ({ children, id = 'languageSelector', label, ...rest }: MenuButtonProps) => {
+  React.useEffect(() => {
+    React.Children.forEach(children, (child: React.ReactElement) => {
+      if (!child.props.lang) {
+        // eslint-disable-next-line no-console
+        console.warn(`NavigationLanguageSelector item "${child.props.label}" is missing a lang property.`);
```

Comment:
Should we add the empty array in the dependencies of the useEffect, so that this warning won't be printed many times? Without the empty array useEffect is run after each render I think?

Reply:
Good catch 👍 
I will add an empty array

Reply:
I will probably need add children as dependency but I think it is ok